lib/faraday_middleware/response/follow_redirects.rb

```diff
@@ -98,6 +109,8 @@ def update_env(env, request_body, response)
         env[:body] = request_body
       end
 
+      env = clear_authorization_header(env, redirect_from_url, redirect_to_url)
```

Comment:
`env` is an object so there's no need to reassign it. Any change made to it's properties in `clear_authorization_header` will be applied to the object itself

```suggestion
      clear_authorization_header(env, redirect_from_url, redirect_to_url)
```